### PR TITLE
clarify steps for adding 3.12.x feeder to superhub

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: enterprise-cfengine-guide/federated-reporting.markdown [hunk 1]
```diff
@@ -18,10 +18,10 @@ Hubs which hosts report to are called Feeder Hubs.
 
 The hub which collects information from Feeder Hubs is called the Superhub.
 
-If all hubs are version 3.14.0 or higher then Mission Portal can be used to
-configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
-earlier version than 3.14.0 an API and manual steps must be followed. This
-documentation will be provided soon.
+If all hubs are version 3.14.0 or higher then Mission Portal can be used to
```
Comment (nickanderson) [inline enterprise-cfengine-guide/federated-reporting.markdown]: Trailing space

File: enterprise-cfengine-guide/federated-reporting.markdown [hunk 2]
```diff
@@ -18,10 +18,10 @@ Hubs which hosts report to are called Feeder Hubs.
 
 The hub which collects information from Feeder Hubs is called the Superhub.
 
-If all hubs are version 3.14.0 or higher then Mission Portal can be used to
-configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
-earlier version than 3.14.0 an API and manual steps must be followed. This
-documentation will be provided soon.
+If all hubs are version 3.14.0 or higher then Mission Portal can be used to 
+configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
```
Comment (nickanderson) [inline enterprise-cfengine-guide/federated-reporting.markdown]: Trailing space

File: enterprise-cfengine-guide/federated-reporting.markdown [hunk 3]
```diff
@@ -168,14 +171,15 @@ $ curl -k -i -s -X POST -u admin:$PASSWORD https://$SUPERHUB/api/fr/setup-hub/su
 $ curl -k -i -s -X POST -u admin:$PASSWORD https://$FEEDER/api/fr/setup-hub/feeder
 ```
 
-### Enable feeder, without API
+### Enable feeder manually
 
 For older hubs:
 
 ```console
 $ ssh $CLOUDUSER$FEEDER
```
Comment (nickanderson) [inline enterprise-cfengine-guide/federated-reporting.markdown]: Is there supposed to be an @ between user and feeder?

File: enterprise-cfengine-guide/federated-reporting.markdown [hunk 4]
```diff
@@ -168,14 +171,15 @@ $ curl -k -i -s -X POST -u admin:$PASSWORD https://$SUPERHUB/api/fr/setup-hub/su
 $ curl -k -i -s -X POST -u admin:$PASSWORD https://$FEEDER/api/fr/setup-hub/feeder
 ```
 
-### Enable feeder, without API
+### Enable feeder manually
```
Comment (vpodzime) [inline enterprise-cfengine-guide/federated-reporting.markdown]: I think the original title makes more sense. *without API* or *only using policy* makes most sense to me.

File: enterprise-cfengine-guide/federated-reporting.markdown (hunk 3 above)
Comment (vpodzime) [reply enterprise-cfengine-guide/federated-reporting.markdown]: `CLOUDUSER=root@`

**Comment by craigcomstock** [comment]
@nickanderson @vpodzime your comments have been addressed. Please re-review. :+1:

**Pull request merged**